### ISSUE TITLE
reviewer: DT as head feature?

### ISSUE THREAD

**Issue by stefan11** [stefan11/german-sentence#30]
> p. 127
> Does it make sense to say that a discourse topic is a head property
> of syntactic categories? (if the answer is no, the DT feature must
> be regarded a convenient technical hack)

I guess it is. We need it at the top level of the phrase. To do.

**Comment by herrjasper** [comment]
The discourse topic itself is not is a head property of a verb. What is encoded as a head property is the verb's preference for one of its arguments to be realized as a topic.

**Comment by stefan11** [comment]
Yes, but the question is whether this should be a HEAD feature. Maybe this is a matter of taste. It is not really syntactic, isn't it?
